www/src/pages/about.mdx

```diff
@@ -0,0 +1,28 @@
+# About
+<p className="lead">
+	Get to know more about the team maintaining React-Bootstrap. Learn a 
+	little history of how, why and when the project started and how
+	you can be a part of it.
+</p>
+
+### Team
+React-Bootstrap is maintained by a [team of developers](https://github.com/orgs/react-bootstrap/people) on Github. We have a growing team
+and if you are interested in re-building the most popular front-end framework with React we would love to hear from you.
+
+### Contributors
+We welcome community support with both pull requests and reporting bugs. Please don't hesitate to jump in.
+Join our growing list of [contributors](https://github.com/react-bootstrap/react-bootstrap/graphs/contributors).
+
+### Sponsorships
+We currently aren't accepting any funding what so ever, but that could change 😉
+
+### Get Involved
+Get involved with React-Bootstrap [by opening an issue](https://github.com/react-bootstrap/react-bootstrap/issues/new) or submitting a pull request.
+See our [contributing guidelines](https://github.com/react-bootstrap/react-bootstrap/blob/master/CONTRIBUTING.md) here.
+
+### External Links
+- [Bootstrap](https://getbootstrap.com/)
+- [React](https://reactjs.org/)
+- [React-router-bootstrap](https://github.com/react-bootstrap/react-router-bootstrap)
```

Comment:
```suggestion
- [React Router Bootstrap](https://github.com/react-bootstrap/react-router-bootstrap)
```
hmm, not entirely sure if it should be `React Router Bootstrap` or `react-router-bootstrap`. The former seems to make more sense for consistency purposes

Reply:
Alright! Changes made.